### DISCUSSION TITLE
Option to blur unread covers

### DISCUSSION THREAD

**Discussion by gregoryn22** [gotson/komga#1549] [Ideas]
### Describe your suggested feature

An unfortunate reality is that sometimes covers of future volumes/issues can be spoilery! I think a useful option to have would be to give a user the option to blur unread covers, ideally on a series-by-series basis.

For example, having not read any volumes/issues in a series, it would look as follows:

![image](https://github.com/gotson/komga/assets/56735924/810a4108-47da-4296-8f5e-2d3cbcf1fe08)

And after reading, for example, volumes/issues 1 and 2, the covers of both read and On Deck volumes/issues would look as follows:

![image](https://github.com/gotson/komga/assets/56735924/e864e1b0-c1d6-482d-884b-f2381d182d54)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
It's being added to per-user settings:
<img width="685" alt="image" src="https://github.com/user-attachments/assets/83088144-c627-4d97-b1f9-8f9074172759" />

It will also blur series poster if all books in the series are unread.

**Reply by gotson** [reply]
🎉 This issue has been resolved in `1.20.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.20.0))
